### PR TITLE
Add remaining missing rules

### LINKED ISSUE #11: Add new rules released since we created this configuration repository
The following rules have been added to eslint and the plugins we use, and we should integrate them into our configuration:

*this list was generated using output from `npm run test`*

* [accessor-pairs](https://eslint.org/docs/rules/accessor-pairs)
* [array-bracket-newline](https://eslint.org/docs/rules/array-bracket-newline)
* [array-element-newline](https://eslint.org/docs/rules/array-element-newline)
* [arrow-body-style](https://eslint.org/docs/rules/arrow-body-style)
* [arrow-parens](https://eslint.org/docs/rules/arrow-parens)
* [arrow-spacing](https://eslint.org/docs/rules/arrow-spacing)
* [block-scoped-var](https://eslint.org/docs/rules/block-scoped-var)
* [block-spacing](https://eslint.org/docs/rules/block-spacing)
* [capitalized-comments](https://eslint.org/docs/rules/capitalized-comments)
* [comma-dangle](https://eslint.org/docs/rules/comma-dangle)
* [default-case-last](https://eslint.org/docs/rules/default-case-last)
* [default-param-last](https://eslint.org/docs/rules/default-param-last)
* [func-names](https://eslint.org/docs/rules/func-names)
* [function-call-argument-newline](https://eslint.org/docs/rules/function-call-argument-newline)
* [function-paren-newline](https://eslint.org/docs/rules/function-paren-newline)
* [generator-star-spacing](https://eslint.org/docs/rules/generator-star-spacing)
* [grouped-accessor-pairs](https://eslint.org/docs/rules/grouped-accessor-pairs)
* [guard-for-in](https://eslint.org/docs/rules/guard-for-in)
* [id-denylist](https://eslint.org/docs/rules/id-denylist)
* [id-length](https://eslint.org/docs/rules/id-length)
* [id-match](https://eslint.org/docs/rules/id-match)
* [implicit-arrow-linebreak](https://eslint.org/docs/rules/implicit-arrow-linebreak)
* [init-declarations](https://eslint.org/docs/rules/init-declarations)
* [jsx-quotes](https://eslint.org/docs/rules/jsx-quotes)
* [keyword-spacing](https://eslint.org/docs/rules/keyword-spacing)
* [lines-around-comment](https://eslint.org/docs/rules/lines-around-comment)
* [lines-between-class-members](https://eslint.org/docs/rules/lines-between-class-members)
* [max-classes-per-file](https://eslint.org/docs/rules/max-classes-per-file)
* [max-lines](https://eslint.org/docs/rules/max-lines)
* [max-lines-per-function](https://eslint.org/docs/rules/max-lines-per-function)
* [max-params](https://eslint.org/docs/rules/max-params)
* [max-statements](https://eslint.org/docs/rules/max-statements)
* [multiline-comment-style](https://eslint.org/docs/rules/multiline-comment-style)
* [multiline-ternary](https://eslint.org/docs/rules/multiline-ternary)
* [newline-per-chained-call](https://eslint.org/docs/rules/newline-per-chained-call)
* [new-parens](https://eslint.org/docs/rules/new-parens)
* [no-await-in-loop](https://eslint.org/docs/rules/no-await-in-loop)
* [no-bitwise](https://eslint.org/docs/rules/no-bitwise)
* [no-confusing-arrow](https://eslint.org/docs/rules/no-confusing-arrow)
* [no-constructor-return](https://eslint.org/docs/rules/no-constructor-return)
* [no-continue](https://eslint.org/docs/rules/no-continue)
* [node/callback-return](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/exports-style](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/file-extension-in-import](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-callback-literal](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-deprecated-api](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-exports-assign](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-extraneous-import](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-extraneous-require](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-missing-import](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-missing-require](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-process-env](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-process-exit](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-restricted-import](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-restricted-require](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-unpublished-bin](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-unpublished-import](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-unsupported-features/es-builtins](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-unsupported-features/es-syntax](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/no-unsupported-features/node-builtins](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/prefer-global/buffer](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/prefer-global/console](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/prefer-global/process](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/prefer-global/text-decoder](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/prefer-global/text-encoder](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/prefer-global/url](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/prefer-global/url-search-params](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/prefer-promises/dns](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/prefer-promises/fs](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/process-exit-as-throw](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [node/shebang](https://github.com/mysticatea/eslint-plugin-node#-rules)
* [no-div-regex](https://eslint.org/docs/rules/no-div-regex)
* [no-duplicate-imports](https://eslint.org/docs/rules/no-duplicate-imports)
* [no-empty-function](https://eslint.org/docs/rules/no-empty-function)
* [no-extra-bind](https://eslint.org/docs/rules/no-extra-bind)
* [no-extra-parens](https://eslint.org/docs/rules/no-extra-parens)
* [no-implicit-globals](https://eslint.org/docs/rules/no-implicit-globals)
* [no-inline-comments](https://eslint.org/docs/rules/no-inline-comments)
* [no-invalid-this](https://eslint.org/docs/rules/no-invalid-this)
* [no-labels](https://eslint.org/docs/rules/no-labels)
* [no-loop-func](https://eslint.org/docs/rules/no-loop-func)
* [no-loss-of-precision](https://eslint.org/docs/rules/no-loss-of-precision)
* [no-magic-numbers](https://eslint.org/docs/rules/no-magic-numbers)
* [no-mixed-operators](https://eslint.org/docs/rules/no-mixed-operators)
* [no-multi-assign](https://eslint.org/docs/rules/no-multi-assign)
* [no-negated-condition](https://eslint.org/docs/rules/no-negated-condition)
* [no-nested-ternary](https://eslint.org/docs/rules/no-nested-ternary)
* [no-new](https://eslint.org/docs/rules/no-new)
* [no-new-wrappers](https://eslint.org/docs/rules/no-new-wrappers)
* [no-nonoctal-decimal-escape](https://eslint.org/docs/rules/no-nonoctal-decimal-escape)
* [no-plusplus](https://eslint.org/docs/rules/no-plusplus)
* [no-promise-executor-return](https://eslint.org/docs/rules/no-promise-executor-return)
* [no-restricted-exports](https://eslint.org/docs/rules/no-restricted-exports)
* [no-restricted-globals](https://eslint.org/docs/rules/no-restricted-globals)
* [no-restricted-imports](https://eslint.org/docs/rules/no-restricted-imports)
* [no-restricted-properties](https://eslint.org/docs/rules/no-restricted-properties)
* [no-restricted-syntax](https://eslint.org/docs/rules/no-restricted-syntax)
* [no-return-assign](https://eslint.org/docs/rules/no-return-assign)
* [no-return-await](https://eslint.org/docs/rules/no-return-await)
* [no-script-url](https://eslint.org/docs/rules/no-script-url)
* [no-shadow](https://eslint.org/docs/rules/no-shadow)
* [no-tabs](https://eslint.org/docs/rules/no-tabs)
* [no-template-curly-in-string](https://eslint.org/docs/rules/no-template-curly-in-string)
* [no-ternary](https://eslint.org/docs/rules/no-ternary)
* [no-undefined](https://eslint.org/docs/rules/no-undefined)
* [no-undef-init](https://eslint.org/docs/rules/no-undef-init)
* [no-underscore-dangle](https://eslint.org/docs/rules/no-underscore-dangle)
* [no-unreachable-loop](https://eslint.org/docs/rules/no-unreachable-loop)
* [no-unsafe-optional-chaining](https://eslint.org/docs/rules/no-unsafe-optional-chaining)
* [no-useless-backreference](https://eslint.org/docs/rules/no-useless-backreference)
* [no-useless-computed-key](https://eslint.org/docs/rules/no-useless-computed-key)
* [no-useless-constructor](https://eslint.org/docs/rules/no-useless-constructor)
* [no-useless-rename](https://eslint.org/docs/rules/no-useless-rename)
* [no-var](https://eslint.org/docs/rules/no-var)
* [no-void](https://eslint.org/docs/rules/no-void)
* [object-curly-newline](https://eslint.org/docs/rules/object-curly-newline)
* [object-property-newline](https://eslint.org/docs/rules/object-property-newline)
* [object-shorthand](https://eslint.org/docs/rules/object-shorthand)
* [one-var](https://eslint.org/docs/rules/one-var)
* [one-var-declaration-per-line](https://eslint.org/docs/rules/one-var-declaration-per-line)
* [operator-assignment](https://eslint.org/docs/rules/operator-assignment)
* [operator-linebreak](https://eslint.org/docs/rules/operator-linebreak)
* [padded-blocks](https://eslint.org/docs/rules/padded-blocks)
* [padding-line-between-statements](https://eslint.org/docs/rules/padding-line-between-statements)
* [prefer-arrow-callback](https://eslint.org/docs/rules/prefer-arrow-callback)
* [prefer-const](https://eslint.org/docs/rules/prefer-const)
* [prefer-destructuring](https://eslint.org/docs/rules/prefer-destructuring)
* [prefer-exponentiation-operator](https://eslint.org/docs/rules/prefer-exponentiation-operator)
* [prefer-named-capture-group](https://eslint.org/docs/rules/prefer-named-capture-group)
* [prefer-numeric-literals](https://eslint.org/docs/rules/prefer-numeric-literals)
* [prefer-object-spread](https://eslint.org/docs/rules/prefer-object-spread)
* [prefer-promise-reject-errors](https://eslint.org/docs/rules/prefer-promise-reject-errors)
* [prefer-regex-literals](https://eslint.org/docs/rules/prefer-regex-literals)
* [prefer-rest-params](https://eslint.org/docs/rules/prefer-rest-params)
* [prefer-spread](https://eslint.org/docs/rules/prefer-spread)
* [prefer-template](https://eslint.org/docs/rules/prefer-template)
* [react/boolean-prop-naming](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/button-has-type](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/default-props-match-prop-types](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/destructuring-assignment](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/display-name](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/forbid-component-props](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/forbid-dom-props](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/forbid-elements](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/forbid-foreign-prop-types](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/forbid-prop-types](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/function-component-definition](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/jsx-boolean-value](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-child-element-spacing](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-closing-bracket-location](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-closing-tag-location](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-curly-brace-presence](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-curly-newline](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-curly-spacing](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-equals-spacing](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-filename-extension](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-first-prop-new-line](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-fragments](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-handler-names](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-indent](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-indent-props](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-key](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-max-depth](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-max-props-per-line](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-newline](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-no-comment-textnodes](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-no-duplicate-props](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-no-literals](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-no-script-url](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-no-target-blank](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-no-undef](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-no-useless-fragment](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-one-expression-per-line](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-pascal-case](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-props-no-multi-spaces](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-props-no-spreading](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-sort-default-props](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-sort-props](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-tag-spacing](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-uses-react](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-uses-vars](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/jsx-wrap-multilines](https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules)
* [react/no-access-state-in-setstate](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-adjacent-inline-elements](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-array-index-key](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-children-prop](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-danger](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-danger-with-children](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-deprecated](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-did-mount-set-state](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-direct-mutation-state](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-find-dom-node](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-is-mounted](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-multi-comp](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-redundant-should-component-update](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-render-return-value](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-set-state](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-string-refs](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-this-in-sfc](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-typos](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-unescaped-entities](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-unsafe](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-unused-state](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/no-will-update-set-state](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/prefer-es6-class](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/prefer-read-only-props](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/prefer-stateless-function](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/require-default-props](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/require-optimization](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/require-render-return](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/self-closing-comp](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/sort-comp](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/sort-prop-types](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/state-in-constructor](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/static-property-placement](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/style-prop-object](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [react/void-dom-elements-no-children](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules)
* [require-atomic-updates](https://eslint.org/docs/rules/require-atomic-updates)
* [require-await](https://eslint.org/docs/rules/require-await)
* [require-unicode-regexp](https://eslint.org/docs/rules/require-unicode-regexp)
* [rest-spread-spacing](https://eslint.org/docs/rules/rest-spread-spacing)
* [sort-imports](https://eslint.org/docs/rules/sort-imports)
* [sort-keys](https://eslint.org/docs/rules/sort-keys)
* [sort-vars](https://eslint.org/docs/rules/sort-vars)
* [space-before-function-paren](https://eslint.org/docs/rules/space-before-function-paren)
* [spaced-comment](https://eslint.org/docs/rules/spaced-comment)
* [symbol-description](https://eslint.org/docs/rules/symbol-description)
* [template-curly-spacing](https://eslint.org/docs/rules/template-curly-spacing)
* [template-tag-spacing](https://eslint.org/docs/rules/template-tag-spacing)
* [vars-on-top](https://eslint.org/docs/rules/vars-on-top)
* [wrap-regex](https://eslint.org/docs/rules/wrap-regex)
* [yield-star-spacing](https://eslint.org/docs/rules/yield-star-spacing)

### PR DESCRIPTION
**Pull request by Avaq**
To close #11

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -79,8 +77,41 @@ module.exports = {
     'max-depth': [2, {max: 5}],
     'max-nested-callbacks': [2, {max: 5}],
     'complexity': [2, {max: 12}],
+    'func-names': [2, 'as-needed'],
```
Comment (Avaq) [inline index.js:80]: Not sure about func names if it means we need to name all our lambdas

File: index.js [hunk 2]
```diff
@@ -101,8 +135,94 @@ module.exports = {
...
+    'space-before-function-paren': [2, 'never'],
+    'spaced-comment': [2, 'always'],
+
+    // Unused rules.
+    'block-scoped-var': [0],
+    'id-denylist': [0],
+    'id-length': [0],
+    'id-match': [0],
+    'max-classes-per-file': [0],
+    'max-lines': [0],
```
Comment (odididi) [inline index.js:190]: we could have a max here and disable it when needed. I would propose maybe 400?

File: index.js [hunk 3]
```diff
@@ -101,8 +135,94 @@ module.exports = {
...
-    'semi-style': [2, 'last']
+    'semi-style': [2, 'last'],
+    'array-bracket-newline': [2, 'consistent'],
+    'array-element-newline': [2, 'consistent'],
+    'block-spacing': [2, 'always'],
+    'capitalized-comments': [2, 'always', {
+      ignoreConsecutiveComments: true,
+      ignoreInlineComments: true,
+    }],
+    'comma-dangle': [2, {
```
[the remark this answers is not in the document]
Comment (Avaq) [reply index.js:143]: The dangling commas on newlines make the diff cleaner when appending to a data structure.

File: index.js (hunk 2 above)
Comment (Avaq) [reply index.js:190]: I guess it's a good number to give the developer a little prod at that point.

File: index.js [hunk 4]
```diff
@@ -101,8 +135,94 @@ module.exports = {
...
+    'multiline-comment-style': [2, 'separate-lines'],
+    'new-parens': [2, 'never'],
+    'newline-per-chained-call': [2, {ignoreChainWithDepth: 3}],
+    'no-inline-comments': [2],
+    'grouped-accessor-pairs': [2],
+    'no-tabs': [2],
+    'object-curly-newline': [2, {consistent: true, minProperties: 0}],
+    'object-shorthand': [2, 'consistent-as-needed'],
+    'one-var': [2, 'never'],
+    'operator-assignment': [2, 'always'],
```
[the remark this answers is not in the document]
Comment (Avaq) [reply index.js:178]: I don't really care: I tend to avoid mutation anyway.

[the remark this answers is not in the document]
Comment (Avaq) [reply index.js:178]: I guess `never` is a bit nicer. `foo = foo + bar` is more explicit than `foo += bar`

File: index.js [hunk 5]
```diff
@@ -101,8 +135,94 @@ module.exports = {
...
+      allowArrayEnd: false,
+    }],
+    'lines-between-class-members': [2, 'always'],
+    'multiline-comment-style': [2, 'separate-lines'],
+    'new-parens': [2, 'never'],
+    'newline-per-chained-call': [2, {ignoreChainWithDepth: 3}],
+    'no-inline-comments': [2],
+    'grouped-accessor-pairs': [2],
+    'no-tabs': [2],
+    'object-curly-newline': [2, {consistent: true, minProperties: 0}],
```
[the remark this answers is not in the document]
Comment (Avaq) [reply index.js:175]: No, setting the minProperties to 0 prevents errors for those cases.

**Review by odididi — APPROVED** [review]

File: index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (d-eisenga) [reply index.js:143]: I vote `always-multiline`

File: .eslintrc.json [hunk 6]
```diff
@@ -1,4 +1,7 @@
 {
   "root": true,
-  "extends": ["./index.js", "./node.js"]
+  "extends": ["./index.js", "./node.js"],
+  "rules": {
+    "no-magic-numbers": [0]
+  }
```
Comment (d-eisenga) [inline .eslintrc.json:6]: This should not be needed anymore, right?

**Review by d-eisenga — DISMISSED** [review]

**Review by d-eisenga — APPROVED** [review]

**Pull request merged**